content/app/development/logic/validation/_index.en.md

```diff
@@ -554,7 +552,7 @@ and `validateRow` only runs validation on the row the user is trying to save. Ex
   "data": {
     "layout": [
       {
-        "id": "demo-gruppe",
+        "id": "demo-gruppe",|
```

Comment:
was this `|` added by mistake?

Reply:
Nice catch! :sweat_smile: I'll fix it immediately